### PR TITLE
Text app search.

### PR DESCRIPTION
**Pull request by KonstantinTsykulenko**

### CODE REVIEW

**Comment by KonstantinTsykulenko** [comment]
>please check new library Serhii created to fix the problem with same configuration objects.
Will do

File: src/main/scala/com/apptentive/appstore/v2/elasticsearch/ESClient.scala [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import io.searchbox.client.config.HttpClientConfig
+import io.searchbox.core.Suggest
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{ExecutionContext, Future}
+
+class ESClient(implicit ec: ExecutionContext) extends LazyLogging {
+
+  private val appIndex = "004-store_apps_v2"
+  private val store_app = "store_app"
```
Comment (ahawker) [inline src/main/scala/com/apptentive/appstore/v2/elasticsearch/ESClient.scala]: Is this used anywhere?

Comment (KonstantinTsykulenko) [reply src/main/scala/com/apptentive/appstore/v2/elasticsearch/ESClient.scala]: Not right now. Are you seeing ES issues? I believe the index is being populated by http://jenkins.corp.apptentive.com/job/appfigures-daily-reviews/

Comment (lozzenko) [reply src/main/scala/com/apptentive/appstore/v2/elasticsearch/ESClient.scala]: It is a reserved word for our international autocomplete index.

Comment (ahawker) [reply src/main/scala/com/apptentive/appstore/v2/elasticsearch/ESClient.scala]: Yeah but specifying it in the code is here is doing _nothing_, right? Is there magic or a dependency somewhere that's using it?

Comment (lozzenko) [reply src/main/scala/com/apptentive/appstore/v2/elasticsearch/ESClient.scala]: Finally got it. You are talking about `store_app`, not `appIndex` variable.
@KonstantinTsykulenko could you please check if we need to inject this document type variable in line 44

Comment (KonstantinTsykulenko) [reply src/main/scala/com/apptentive/appstore/v2/elasticsearch/ESClient.scala]: No, we don't, it's a leftover from initial implementation. I'll remove it.

**Pull request merged**